person_capture/gui_app.py

```
@@ -2687,6 +2709,25 @@ def _cooldown_frames() -> int:
+                try:
+                    os.environ["OPENCV_FFMPEG_CAPTURE_OPTIONS"] = "hwaccel;cuda"
+                    reader = cv2.VideoCapture(video_path, cv2.CAP_FFMPEG)
+                finally:
+                    if prev_ffmpeg_opts is None:
+                        os.environ.pop("OPENCV_FFMPEG_CAPTURE_OPTIONS", None)
+                    else:
+                        os.environ["OPENCV_FFMPEG_CAPTURE_OPTIONS"] = prev_ffmpeg_opts
+                try:
+                    reader.set(cv2.CAP_PROP_HW_ACCELERATION, 1)
+                except Exception:
+                    pass
```

Ruff (0.15.11)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)